frontend/src/components/phones/onboarding/RelayNumberPicker.tsx

```diff
@@ -90,39 +100,38 @@ const RelayNumberSelection = (props: RelayNumberSelectionProps) => {
   const [relayNumberSuggestions, setRelayNumberSuggestions] = useState<
     string[]
   >([]);
-  const [sliceStart, setSliceStart] = useState(0);
-  const [sliceEnd, setSliceEnd] = useState(3);
+  const [relayNumberIndex, setRelayNumberIndex] = useState(0);
 
   useEffect(() => {
-    (async () => {
-      setIsLoading(true);
-      const resp = await getRelayNumberSuggestions();
-      setIsLoading(false);
-      const respObject = await resp.json();
-      const suggestedNumbers: string[] = [];
-
-      // TODO: DRY If statements below
-      if (respObject.other_areas_options) {
-        for (const otherAreaOption of respObject.other_areas_options) {
-          suggestedNumbers.push(otherAreaOption.phone_number);
-        }
-      }
+    _getRelayNumberSuggestions();
+  }, []);
 
-      if (respObject.same_area_options) {
-        for (const sameAreaOption of respObject.same_area_options) {
-          suggestedNumbers.push(sameAreaOption.phone_number);
-        }
-      }
+  // GET request to get relay number suggestions
+  const _getRelayNumberSuggestions = () => {
```

Comment:
@Vinnl @groovecoder I'm not great at names... let me know if you have something better.

Reply:
I think what I'd do here is replace the original `getRelayNumberSuggestions` in `/frontend/src/hooks/api/relayNumber.ts` with an SWR hook just like we do for other API endpoints, e.g. something like

```ts
export function useRelayNumberSuggestions(): SWRResponse<RelayNumberOption[], unknown> {
  const relayNumberSuggestions = useApiV1("/relaynumber/suggestions/");
  return relayNumberSuggestions;
}
```

Then you can simply include that hook in this components, and it will make the request as soon as it renders, without you having to manually keep track of whether it's loading and all that, nor having to include a wrapper that does roughly the same and hence has a very similar name. Should save a bit of the awkwardness here, I think.

Reply:
Spoke to Luke about this one. He decided to have us just cycle through the initial batch we get from the API. So I'm taking the hook route. Still keeping the spinner though, with Maxx's implementation - I like the implementation - we remove the spinner exactly when we want it to go away. However, if you want me to change it, I can open another PR.